src/Promitor.Tests.Unit/Builders/Metrics/v1/MetricsDeclarationBuilder.cs

```diff
@@ -1022,5 +1050,13 @@ private AzureMetricConfigurationV1 CreateAzureMetricConfiguration(string azureMe
 
             return metricConfig;
         }
+
+        private LogAnalyticsConfigurationV1 CreateLogAnalyticsConfiguration(string query, string interval)
```

Comment:
We should have tests on this as well for the validation.

SCenarios to add:
- Only LA config is provided, LA resource passes
- No LA config is provided, LA resource fails
- No Azure Monitor config is provided, Azyre Monitor resource fails
- Both config are provided, LA resource passes

Reply:
I already put validation tests here https://github.com/tomkerkhove/promitor/blob/3c1e0e68cea31a188fb0aec56b98aeef7e2dc4f5/src/Promitor.Tests.Unit/Validation/Scraper/Metrics/ResourceTypes/LogAnalyticsDeclarationValidationStepsTests.cs